<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>468</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Set Options</string>
  </property>
  <widget class="QPushButton" name="btnLtGrey">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>100</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnYellow">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>140</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnWhite">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>140</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>325</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>325</y>
     <width>44</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Levels</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBlue">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTeal">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDkGrey">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>230</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBrown">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>100</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinScale">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>274</y>
     <width>62</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>2.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>1.000000000000000</double>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDraw">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>274</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Draw</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>274</y>
     <width>37</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Scale</string>
   </property>
  </widget>
  <widget class="QSlider" name="yOffset">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>30</y>
     <width>18</width>
     <height>350</height>
    </rect>
   </property>
   <property name="maximum">
    <number>199</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="sliderPosition">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QFrame" name="selectedColor">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QRadioButton" name="btnCircle">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>190</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </widget>
  <widget class="QLabel" name="offset">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>( -0.50 , 0.25 )</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnViolet">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSlider" name="xOffset">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>380</y>
     <width>371</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>0</number>
   </property>
   <property name="maximum">
    <number>199</number>
   </property>
   <property name="singleStep">
    <number>0</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="sliderPosition">
    <number>100</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPink">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>140</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnPlum">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAqua">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>60</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>274</y>
     <width>37</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Sides</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGreen">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLime">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinLevel">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>325</y>
     <width>49</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximum">
    <number>10</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinSides">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>274</y>
     <width>49</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
  </widget>
  <widget class="QPushButton" name="btnDkRed">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOrange">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinPitch">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>325</y>
     <width>66</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <double>-3.140000000000000</double>
   </property>
   <property name="maximum">
    <double>3.140000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.010000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>241</x>
     <y>325</y>
     <width>35</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Pitch</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBlack">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOpen">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>81</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>311</x>
     <y>119</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>50</y>
     <width>81</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="btnRandom">
      <property name="text">
       <string>Rand</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btnKoch">
      <property name="text">
       <string>Koch</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btnSquare">
      <property name="text">
       <string>Square</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btnPeano">
      <property name="text">
       <string>Peano</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="btnPoly">
      <property name="text">
       <string>Poly</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
